core/src/blocktree_processor.rs

```diff
@@ -942,13 +945,13 @@ pub mod tests {
         // Check all accounts are unlocked
         let txs1 = &entry_1_to_mint.transactions[..];
         let txs2 = &entry_2_to_3_mint_to_1.transactions[..];
-        let locked_accounts1 = bank.lock_accounts(txs1);
+        let locked_accounts1 = bank.lock_accounts(txs1, None);
         for result in locked_accounts1.locked_accounts_results() {
             assert!(result.is_ok());
         }
         // txs1 and txs2 have accounts that conflict, so we must drop txs1 first
         drop(locked_accounts1);
-        let locked_accounts2 = bank.lock_accounts(txs2);
+        let locked_accounts2 = bank.lock_accounts(txs2, None);
```

Comment:
To test the above changes, you can add a test that generates some entries that create, then touch the same accounts. 

For instance:
1) Create accounts A, B (see examples in tests of `system_transaction::create_user_account`) with N tokens in each of those accounts
2) Generate a set of entries that contains first a `transfer` of `N` tokens  A -> B, and second, a transfer of `2N -1` tokens from B-> A. 
3)Then run multiple iterations of `par_execute_entries` on different banks with this same set of entries, and ensure that on each run, the final balance is A: 2N -1 tokens, B: 1 token.

Reply:
cool. Will add it.

Reply:
@carllin On second thought, along with that I will add a test that proves that for an entry, multiple different ordering of tx execution, produces same result.

Reply:
@ParthDesai, you're right :D, that's the correct test. I misunderstood the area of randomization.